tests/src/dir.common.props

```diff
@@ -27,13 +27,13 @@
 
 <!-- Setup the default output and intermediate paths -->
   <PropertyGroup>
-    <BaseOutputPathWithConfig>$(ProjectDir)\..\binaries\tests\$(Platform)\$(Configuration)\</BaseOutputPathWithConfig>
+    <BaseOutputPathWithConfig>$(ProjectDir)\..\binaries\tests\$(Platform).$(OS).$(Configuration)\</BaseOutputPathWithConfig>
```

Comment:
Gotcha.  I think that just changed as well.  When I added OS into corefx it was in the middle.